lib/index.js

```diff
@@ -46,15 +46,14 @@ module.exports = {
     'consistent-this': [2, 'self'],
     'new-parens': 2,
     'no-array-constructor': 2,
-    'no-arrow-condition': 2,
+    'no-confusing-arrow': 2,
     'no-new-object': 2,
     'no-spaced-func': 2,
     'no-mixed-spaces-and-tabs': 2,
-    'space-after-keywords': 2,
+    'keyword-spacing': 2,
```

Comment:
Hm, what I really want is to ignore the before setting, as we weren't using it before. I'm trying to think of examples in the hapi ecosystem that would break in each case (`before` as `true` and `false`).